### PR TITLE
Suggested updates for #167

### LINKED ISSUE #167: Update readme for require statements..,.
Replace `require('jackrabbit');` with `require('@pager/jackrabbit');`

### PR DESCRIPTION
**Pull request by tracker1**
* Modify readme to show proper requires statement(s),
* Add additional keywords for package visibility
* Update author/contributors
* Bump revision for publish

Closes #167

### CODE REVIEW

**Comment by cilindrox** [comment]
Thanks for this @tracker1

File: package.json [hunk 1]
```diff
@@ -18,15 +22,15 @@
...
-    "lint": "eslint lib/** test/**"
-  },
   "engines": {
     "node": ">= 10.x"
   },
-  "author": "Hunter Loftis <hunter@hunterloftis.com>",
   "license": "MIT",
+  "author": "Pager Developers <engeneering@pager.com>",
+  "contributors": [
+    "Pager Developers <engeneering@pager.com>",
```
Comment (ryanhallcs) [inline package.json]:
```suggestion
    "Pager Developers <engineering@pager.com>",
```

File: package.json [hunk 2]
```diff
@@ -18,15 +22,15 @@
...
-  "scripts": {
-    "test": "mocha",
-    "lint": "eslint lib/** test/**"
-  },
   "engines": {
     "node": ">= 10.x"
   },
-  "author": "Hunter Loftis <hunter@hunterloftis.com>",
   "license": "MIT",
+  "author": "Pager Developers <engeneering@pager.com>",
```
Comment (ryanhallcs) [inline package.json]:
```suggestion
  "author": "Pager Developers <engineering@pager.com>",
```

Comment (cilindrox) [reply package.json]: Good catch - thanks @ryanhallcs

**Comment by tracker1** [comment]
@cilindrox @ryanhallcs thanks for catching the fat-key on the email address, committed as suggested.

**Pull request merged**